CollAction/Controllers/ProjectsController.cs

```diff
@@ -362,5 +362,35 @@ public ViewResult StartInfo()
         {       
             return Json(await _context.Categories.Select(c => new { c.Id, c.Name }).ToListAsync());
         }
+
+        [HttpGet]
+        public async Task<JsonResult> GetTileProjects(int? categoryId, int? locationId)
+        {
+            var tileProjects = await _service.GetTileProjects(p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted
+            && (categoryId != null ? p.CategoryId == categoryId : true) && (locationId != null ? p.LocationId == locationId : true));
+
+            List<DisplayTileProjectViewModel> displayTileProjectViewModel = tileProjects.Select(t =>
+            {
+                var tileViewModel = new DisplayTileProjectViewModel
+                {
+                    ProjectId = t.ProjectId,
+                    ProjectName = t.ProjectName,
+                    ProjectProposal = t.ProjectProposal,
+                    CategoryName = t.CategoryName,
+                    CategoryColorHex = t.CategoryColorHex,
+                    LocationName = t.LocationName,
+                    Target = t.Target,
+                    Participants = t.Participants,
+                    ProgressPercent = t.ProgressPercent,
+                    BannerImagePath = (t.BannerImage != null) ? Url.Content(t.BannerImage.Filepath.Replace('\\', '/')) : "https://placeholdit.imgix.net/~text?txtsize=33&txt=Project%20Image"
```

Comment:
I have no idea about aspnet but should this banner selection code go in the ViewModel?